guides/common/modules/proc_creating-an-activation-key.adoc

```diff
@@ -11,32 +11,17 @@ endif::[]
 Subscription Manager attaches subscriptions differently depending on the following factors:
 
 * Are there any subscriptions associated with the activation key?
-* Is the auto-attach option enabled?
 * For {RHEL} 8 hosts: Is there system purpose set on the activation key?
 
 Note that {Project} automatically attaches subscriptions only for the products installed on a host.
-For subscriptions that do not list products installed on {RHEL} by default, such as the Extended Update Support (EUS) subscription, use an activation key specifying the required subscriptions and with the auto-attach disabled.
+For subscriptions that do not list products installed on {RHEL} by default, such as the Extended Update Support (EUS) subscription, use an activation key specifying the required subscriptions.
 
-Based on the previous factors, there are three possible scenarios for subscribing with activation keys:
+Based on the previous factors, there is one possible scenario for subscribing with activation keys:
```

Comment:
All of this can be removed. Subscriptions are no longer attached to activation keys.